writers/mongodb/mongodb.go

```diff
@@ -0,0 +1,49 @@
+package mongodb
+
+import (
+	"context"
+
+	"github.com/mongodb/mongo-go-driver/bson"
+	"github.com/mongodb/mongo-go-driver/mongo"
+
+	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/writers"
+)
+
+const (
+	collectionName string = "mainflux"
+)
+
+var _ writers.MessageRepository = (*mongoRepo)(nil)
+
+type mongoRepo struct {
+	db *mongo.Database
+}
+
+// New returns new MongoDB writer.
+func New(db *mongo.Database) (writers.MessageRepository, error) {
+	return &mongoRepo{db}, nil
+}
+
+func (repo *mongoRepo) Save(msg mainflux.Message) error {
+	coll := repo.db.Collection(collectionName)
+
+	_, err := coll.InsertOne(
+		context.Background(),
+		bson.NewDocument(
```

Comment:
Is there a reason that we write field-by-field? Isn't Mongo driver smart enough that if we how whole structure at it, it will figure out the fields?